README.md

```diff
@@ -105,6 +105,48 @@ cfbs clean
 cfbs input create-single-file
 ```
 
+This command prompts the user for module input to configure what the module should do.
+
+Only works for modules which accept input, indicated by the `"input"` key.
+This `input` key contains the specification for what input the module accepts, if any.
+
+User's responses are stored in `<module-name>/input.json`.
```

Comment:
I still really do not like this. It makes it difficult to tell what's cfbs managed and what's my own stuff in the cfbs project. I thin this would be much better off under a common top level directory like `cfbs-module-input/<module-name>/input.json`

Reply:
This is the default, we can discuss adding the possibility to customize it: https://tracker.mender.io/browse/CFE-4091